src/extension.ts

```diff
@@ -16,26 +15,26 @@ const cmds = {
  * This method is called when the extension is activated.
  * @param context The extension context
  */
-export function activate(context: ExtensionContext) {
+export function activate(context: ExtensionContext): void {
 	for (const cmd in cmds) {
-		const handler = cmds[cmd];
+		const handler = cmds[cmd] as () => void;
 		const disposable = vscode.commands.registerCommand(cmd, handler);
 		context.subscriptions.push(disposable);
 	}
 	const isVlsEnabled = getWorkspaceConfig().get<boolean>('vls.enable');
-	
+
 	workspace.onDidChangeConfiguration((e: ConfigurationChangeEvent) => {
 		if (e.affectsConfiguration('v.vls.enable')) {
 			const isVlsEnabled = getWorkspaceConfig().get<boolean>('vls.enable');
 			if (isVlsEnabled) {
-				activateVls(context);
+				void activateVls(context);
 			} else {
-				deactivateVls();
+				void deactivateVls();
 			}
 		}
 	})
 
 	if (isVlsEnabled) {
-		activateVls(context)
+		void activateVls(context)
```

Comment:
statement with type? never saw that :D

Reply:
A short search showed that you should prefix calls to async functions with `void` when their result is not used.